torch/__init__.py

```diff
@@ -633,6 +633,11 @@ def set_default_device(device):
         is causing problems for you, please comment on
         https://github.com/pytorch/pytorch/issues/92701
 
+    .. note::
+
+        This doesn't affect functions that work with shared memory, like: 
```

Comment:
I think we should avoid the term shared memory here as this could be conflated with [`Tensor.shared_memory`](https://pytorch.org/docs/stable/generated/torch.Tensor.share_memory_.html), which I don't think is the meaning here. 

nit: 
```suggestion
        This doesn't affect functions that create tensors that share the same memory as the input, like: 
```

Reply:
Good point